Enhance Code Naming and Readability on Class

haritsfahreza opened the pull request:
Rename test method name on given and result part, to make it easier to
understand.
Create Plateu class to keep maximum X and Y.
Breakdown runCommands method to turnRight, turnLeft and moveForward
methods, to make easier to read.

haritsfahreza committed:
Enhance Code Naming and Readability on Class

Rename test method name on given and result part, to make it easier to
understand.
Create Plateu class to keep maximum X and Y.
Breakdown runCommands method to turnRight, turnLeft and moveForward
methods, to make easier to read.

sihendra commented on src/test/java/id/fahreza/marsrover/MarsRoverTest.java:
```diff
@@ -6,128 +6,95 @@
-        MarsRover marsRover = new MarsRover(x, y);
-        Assert.assertEquals(x, marsRover.getMaximumX());
-        Assert.assertEquals(y, marsRover.getMaximumY());
+    public void constructor_givenUpperRightCoordinate_shouldSetMamaximumXimumXY() {
```
Please fix the typo

haritsfahreza replied:
Done, already fixed on the next commit. Please review it again.

haritsfahreza committed:
Fix Plateau and Test Method Typo

Fix Plateu typo to become Plateau
Fix test method name typo because of unclean replacement

sihendra reviewed (commented):
Separate Plateau test class

haritsfahreza committed:
Create PlateauTest and Fix MarsRoverTest method

Create PlateauTest to separate Plateau constructor test
Fix MarsRoverTest method name by change given Plateau
